remoteworkitem/trackeritem_repository.go

```diff
@@ -77,7 +77,7 @@ func lookupIdentities(ctx context.Context, db *gorm.DB, remoteWorkItem RemoteWor
 			}
 			creatorLogin := fieldValue.(string)
 			creatorProfileURL := remoteWorkItem.Fields[remoteCreatorProfileURL].(string)
-			identity, err := identityRepository.Lookup(context.Background(), creatorLogin, creatorProfileURL, providerType)
+			identity, err := identityRepository.Lookup(context.Background(), creatorLogin, creatorProfileURL, tq.TrackerType)
```

Comment:
I'm a bit sceptical for why and how the tracker type corresponds to the provider type in the identity repo. Can you please explain what both are and if it is only by accident that you can use the tracker type for it?